codepropertygraph/src/main/scala/io/shiftleft/passes/DiffGraphProtoSerializer.scala

```diff
@@ -150,12 +148,20 @@ class DiffGraphProtoSerializer {
       .setPropertyName(EdgePropertyName.valueOf(propertyKey))
       .build
 
-  private def nodeProperty(key: String, value: Any) = {
-    CpgStruct.Node.Property
-      .newBuilder()
-      .setName(NodePropertyName.valueOf(key))
-      .setValue(protoValue(value))
-      .build()
+  //fixme
```

Comment:
???

Reply:
:joy: 

It is fixed now, this is a stale memo-at-self from developing this (rather voluminous and coupled) PR. I will remove the comment in the next improvement round of this PR.